hosts/hosts.go

```diff
@@ -56,13 +57,19 @@ func (h *Host) CleanUp() error {
 
 func DeleteNode(toDeleteHost *Host, kubeClient *kubernetes.Clientset) error {
 	logrus.Infof("[hosts] Cordoning host [%s]", toDeleteHost.Address)
-	err := k8s.CordonUncordon(kubeClient, toDeleteHost.HostnameOverride, true)
-	if err != nil {
+	if _, err := k8s.GetNode(kubeClient, toDeleteHost.HostnameOverride); err != nil {
+		if !apierrors.IsNotFound(err) {
```

Comment:
Start with `if apierrors.isNotFound`, and if met, log a messgae `Can't find k8s node by name []`